platform-tests/pom.xml

```diff
@@ -49,8 +43,10 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
         <dl4j.version>1.0.0-SNAPSHOT</dl4j.version>
-        <platform.classifier>${os.name}-${os.arch}</platform.classifier>
+        <platform.classifier>windows-x86_64</platform.classifier>
```

Comment:
Maybe have a profile for this? In principle that should pick up the correct value anyway, so I'm not sure why you need to change it

Reply:
On windows it picks up something like: windows-AMD64 or something off. I'm not sure why. In practice I usually override this when running CI builds. I get what you mean though.